liubov.tarasenko/task1.rb

```diff
@@ -0,0 +1,10 @@
+def matrix
+  arr = 3.times.map{ 3.times.map { rand(-10..10) } }
```

Comment:
@lubo4kaTarasenko your solution looks good. however, there is a better way to create a 2dimention array with random numbers `Array.new(3) { Array.new(3) { rand -10..10 } }`

Reply:
Ok, thanks